pr_testing/test_multiple_prs.sh

```diff
@@ -293,6 +293,15 @@ for U_REPO in $(echo ${UNIQ_REPOS} | tr ' ' '\n'  | grep -v '/cmssw$' ); do
     for PR in ${FILTERED_PRS}; do
         ERR=false
         git_clone_and_merge "$(get_cached_GH_JSON "${PR}")" || ERR=true
+	if [[ $(echo ${PR} | grep "cmsdist") ]]; then  # Check for CRAB updates to trigger unit test
+	    pushd cmsdist
+	    UPDATES=$(git diff origin/${BASE_BRANCH} --name-only)
+            if [[ $(echo ${UPDATES} | grep -E \("crab.*spec"\|"crab.*file"\)) ]]; then
+                echo "There is a CRAB update."
+		touch ${WORKSPACE}/run-crab.prop
```

Comment:
do not create this file here ( in case of build failure we do not want to start crab tests). Just set `DO_CRAB_TESTS=false` at [the start of this script](https://github.com/cms-sw/cms-bot/blob/master/pr_testing/test_multiple_prs.sh#L87) , set `DO_CRAB_TESTS=true` here. Reset to false [here](https://github.com/cms-sw/cms-bot/blob/master/pr_testing/test_multiple_prs.sh#L1051)  if there are build errors.